.changeset/blue-turkeys-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/auth-construct': minor
+---
+
+adding customAttributes into userAttributes
```

Comment:
We prefer to use imperative verb form.

See https://www.gitkraken.com/learn/git/best-practices/git-commit-message#using-imperative-verb-form